cardano-api/src/Cardano/Api/Protocol/Types.hs

```diff
@@ -37,7 +41,8 @@ import qualified Ouroboros.Consensus.Shelley.Ledger.Block as Consensus
 
 class (RunNode blk, IOLike m) => Protocol m blk where
   data ProtocolInfoArgs m blk
```

Comment:
It's not your bailiwick, but FYI to anyone reading: none of the existing `ProtocolInfoArgs` instances use the `m` parameter.